### DISCUSSION TITLE
v1.28 feedback thread

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#92] [Ideas & Feedback]
Highlighted changes:
- added leveling tracker: leveling related features
  - dynamic leveling-guide overlays
  - gear tracker
  - gem search-strings for bulk-buying
- added Overlayke: Lake of Kalandra feature
  - preview tablet changes
  - compare difficulty stats
  - let it count to 10 for you

**Comment by guiltyhollow** [comment]
Is there a possibility to add the map layout into leveling-guide-overlays ?
thank you

**Reply by Lailloken** [reply]
Do you mean the layout of campaign sections or actual endgame maps? The guides generated with exile-leveling include cardinal directions where applicable. This feature is based on a request, and I do not intend to make it full-fledged because there is another tool that explicitly specializes on leveling overlays: https://github.com/JusKillmeQik/PoE-Leveling-Guide. I pursued this request because it provided an opportunity to work with client-generated data, and it was a neat detour to also test other things.
EDIT: I may look into it further in the future if there's demand.

**Comment by PergonGER** [comment]
First of all, thank you for your effort. Now i am thinking about other programming projects that came to my mind but i never started. Good luck on your journey.
Would be great if it would be possible to save different regex (like https://poe.re/#/vendor)
Is there a way to see gem links or set them up?
Or link notepads to certain acts? So i can have my links in a notepad and always the up to date notepad? :-)

**Reply by Lailloken** [reply]
You're welcome, and thanks to you too.

> Would be great if it would be possible to save different regex

Do you mean additional regex, in addition to the gem ones? Yes, there's the search-strings feature you can use to store regex strings, and even separately for vendors and the stash. You can read about setting that up in the wiki.

> Is there a way to see gem links or set them up?

Currently not. The overlay shows everything that is imported from exile-leveling.

> Or link notepads to certain acts? So i can have my links in a notepad and always the up to date notepad?

The notepad has different modes. You could set up a 'grouped' text-widget. It's basically a post it note with multiple pages you can switch between, so you could create a page for every act or something. That's also described in the wiki.

**Comment by Daemios** [comment]
Hi, I'm a little confused by the search-strings feature. I have verified my omni key works (mousing over an item and activating the menu for example is fine), but trying to use the omni key while the vendor window is open doesn't produce the menu as shown on the wiki. Is there some additional setup I have to do? I have two search strings, so it's not that it's trying to fill the search with the single string.

**Reply by Lailloken** [reply]
You have to set up the 'vendor' and 'stash' image-check in the screen-checks settings menu, so the script knows the vendor window is open when you press the omni-key. Since the omni-key is context-sensitive, it checks different parameters when it is pressed (was the cursor hovering over an item, was a certain window open, etc.) and executes the appropriate feature. Takes a bit getting used to but also only needs to be set up once.

**Reply by Daemios** [reply]
Man I really appreciate you responding the night before league launch. Very cool tool, your help made it simple to understand.

**Reply by Lailloken** [reply]
I'm in Central Europe, so you got me right when I came back from the morning walk with the dog. Glad it works and that you like it, and always happy to help out.

**Comment by frysee** [comment]
Just wanted to drop this piece of info here in case someone stumbles across the same problem: When launching LLUI on an outdated version of AHK (1.1.30.x in my case), the script fails with a pretty random error: 
```
Error at line 3730.
Line Text: Switch A_Index
Error: This line does not contain a recognized action.
```
Updating AHK to a more recent version solved the issue for me.

**Reply by Lailloken** [reply]
I meant to add that to general trouble-shooting steps but forgot. Thanks for reminding me.

**Comment by OmikronGamma** [comment]
Hi, hello,
I dont know how feasible, but could you add "support" for craftofexile? There is an "import item" button on that site, what if omni-key could do that? "Redirect" an item to CoE to see whats available for slam/unveil?

**Reply by Lailloken** [reply]
I just took a quick look at the site. There doesn't seem to be a direct way to access the import window via a URL, so the best I can do is add an entry to the omni-key context menu that opens the main page. The omni-key copies advanced item mods ever since I implemented the recombinator calculator, so the user would only have to click the import button on the site and paste the item info. As far as I can tell, once an item has been imported on the main page, one can switch between the different crafting features and reload the imported item.

Let me know what you think and feel free to suggest improvements (I don't use that site, so I'm not sure about the 'work-flow').

**Reply by OmikronGamma** [reply]
I was hoping there would be a way, since this one trade site extension (tft trade extension) allows this kind of thing.

`so the best I can do is add an entry to the omni-key context menu that opens the main page`
Better than nothing.

`As far as I can tell, once an item has been imported on the main page, one can switch between the different crafting features`
Thats true.

**Reply by Lailloken** [reply]
> since this one trade site extension (tft trade extension) allows this kind of thing

I can look into it. I couldn't find any documentation for integration on the site itself, so I just thought it wasn't supported.

**Reply by Lailloken** [reply]
Still can't find any documentation on importing items directly, and importing via the extension also doesn't seem to work on my end. It imports the base-type but without any affixes. Is the extension outdated or is it something on my end? If you know where to find documentation, let me know.

If it is indeed the case that direct-import methods can be outdated, I'd rather stick with manual importing. It may be a bit more work for the user, but this ensures compatibility -- and I want to avoid having to constantly stay in the loop with external projects.

**Reply by OmikronGamma** [reply]
Yes, my bad. Sorry for this misinformation.
I pressed on the "export" button, saw that item base, item level and whatnot imported, but didnt check the affixes.

**Comment by OmikronGamma** [comment]
Didnt want to open an issue, but i noticed that omni-key doesnt work with unidentified items. 
Is this intentional or limitations of game/ahk?
Im on latest 1.28.4 version. Tried old version (1.28.0), same result.

**Reply by Lailloken** [reply]
Which type of unid'ed item do you want to use it on (and which feature)? I think I just blanket-disabled it for unid'ed items because some features had problems (e.g. map-info panel), instead of disabling them for individual features. I could enable it on a case-by-case basis if required.

**Reply by OmikronGamma** [reply]
I was to check on mods on a hubris circlet, which mods require what ilvl. Found some random hubris im stash, pressed omni-key and it didnt work. 
At first i thought it was bug related to it being magic rarity, but turns out unique jewels, for example, do not work too.

**Reply by Lailloken** [reply]
I see. Yeah, I can enable it for non-unique gear. But it won't work with uniques because of base-type overlaps.

**Reply by OmikronGamma** [reply]
Another small thing, this new integration with CoE (thanks for it), doesnt seem to work with cluster jewels.

**Reply by Lailloken** [reply]
Ah right, it's because they require a custom crafting table link, which overrides other links. I'll need to rebuild the omni-key context menu (which I wanted to do soon anyway) because it has basically become a Jenga tower with exclusions and custom rules continually stacked onto it.

Thanks for all the valuable feedback and suggestions you have provided from the very start of the project. Much appreciated.
